docs/roadmap.md

```diff
@@ -42,10 +42,10 @@ via Git repository.
 The feature streamlines the process of adding repositories and clusters to the project and makes it self-service.
 Instead of asking an administrator to change Argo CD settings end users can perform the change independently.
 
-### Headless Argo CD (aka GitOps Agent) ([proposal](https://github.com/argoproj/argo-cd/pull/6385))
+### Core Argo CD (aka GitOps Agent) ([proposal](https://github.com/argoproj/argo-cd/pull/6385))
```

Comment:
Let's remove "GitOps agent". I dont think we want to use that terminology yet.

Reply:
done